strutil/ctrl16.go

```diff
@@ -0,0 +1,51 @@
+// +build !go1.7
```

Comment:
Sorry for a perhaps naive question, why do we need two versions of this file? Can you please add that as a comment.